tests/TestHelpers/WebDavHelper.php

```diff
@@ -458,6 +473,7 @@ public static function getPersonalSpaceIdForUser(string $baseUrl, string $user,
 			// we expect to get a multipart XML response with status 207
 			$status = $response->getStatusCode();
 			if ($status !== 207) {
+				if ($returnFakeIfNotFound) return $fakeId;
```

Comment:
This is dangerous. A not careful reformatting could kick the return out of the if-statement 
```suggestion
				if ($returnFakeIfNotFound) {
 				 return $fakeId;
				 }
```